placedebot.user.js

```diff
@@ -67,7 +67,8 @@ const COLOR_MAPPINGS = {
 		duration: 10000
 	}).showToast();
 
-	setInterval(updateOrders, 5 * 60 * 1000); // Update orders elke vijf minuten.
+	setInterval(updateOrders, 5 * 60 * 1000);	// Update orders every five minutes.
+	setInterval(reloadSite, 60 * 1000 * 60);	// refresh the site every hour, because reddit sometimes disconnects...
```

Comment:
This does not need to be an interval, a timeout is enough :)

Reply:
true, updated.